packages/orchestrator/go.mod

```diff
@@ -44,6 +45,8 @@ require (
 	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
 )
 
+require github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
+
```

Comment:
can you merge this with the group below? dunno why they're separate, but they shouldn't be